modules/core/pom.xml

```diff
@@ -420,7 +427,7 @@
 			<version>${org.wso2.json.version}</version>
 		</dependency>
     </dependencies>
-<properties>
-		<version.orbit.rabbitmq>3.6.6.wso2v1</version.orbit.rabbitmq>
-	</properties>
+    <properties>
+        <version.orbit.rabbitmq>3.6.6.wso2v1</version.orbit.rabbitmq>
```

Comment:
Let's refine version properties in the root pom

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/4b18a1559542be1fb38b759acf1f970cf3ab5e43